python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -945,6 +946,66 @@ def create_array(dtype):
         dtype=dtype)
 
 
+def equal(x, y, cond=None, **ignored):
```

Comment:
`equal` is already in the control flow APIs : https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/control_flow.py#L978

Reply:
@jacquesqiao Thx. I have fixed the conflicts.